### ISSUE TITLE
Can not display on the 2nd monitor

### ISSUE THREAD

**Issue by luozhenyu336** [ValveSoftware/openvr#529]
Hi guys, I wrote a DIY openVR HMD driver base on the driver_sample of openVR SDK 1.06. The driver worked in extend mode, but it can only display on the center of primary monitor , I want to it play on the second monitor. 
Here is what the code i changed:

	virtual bool IsDisplayOnDesktop() 
	{
		return true;
	}

	virtual bool IsDisplayRealDisplay() 
	{
		return true;
	}

default.vrsettings:
      "windowX" : 3840,
      "windowY" : 0,
      "windowWidth" : 1920,
      "windowHeight" : 1080,
      "renderWidth" : 1920,
      "renderHeight" : 1080,
My os is windows 10 Enterprise X64.
I have 2 monitors, the primary one is 3840  2160, the 2nd is 1920 1080, the monitors align like this:
![image](https://cloud.githubusercontent.com/assets/13621537/26479205/1c637360-4204-11e7-966e-6105995d362b.png)
Where is the problem? Is any one have the same problem?

**Comment by JoeLudwig** [comment]
The problem is compositor "debug" mode, which is automatically activated for any driver named "null" or "sample". Rename your driver (the DLL and the directory it lives in) to something else and it will obey GetWindowBounds.

The next SDK will move this to a property so be in or out of debug mode will be under the control of the driver itself.  That change is in the SteamVR Beta now, so if you opt-in to the beta, don't be surprised if the behavior changes.

**Comment by luozhenyu336** [comment]
@JoeLudwig Thank you! I will try it.

**Comment by luozhenyu336** [comment]
@JoeLudwig Thank you for you helping, but it do not work. I changed my driver name to driver_test and hard coding the property like this:
		m_sSerialNumber = "1234";
		m_sSerialNumber = "Model 1";

		m_nWindowX = 3840;
		m_nWindowY = 0;
		m_nWindowWidth = 1920;
		m_nWindowHeight = 1080;
		m_nRenderWidth = 1920;
		m_nRenderHeight = 1080;
		m_flSecondsFromVsyncToPhotons = 0.011;
		m_flDisplayFrequency = 0;

The driver loaded successfully and worked, but it made the primary monitor full screen, not the second one. I saw the log of vrserver.txt, it seemed OK:
Sat May 27 2017 11:52:06.811 - test: Using settings values
Sat May 27 2017 11:52:06.811 - test: driver_null: Serial Number: Model 1
Sat May 27 2017 11:52:06.812 - test: driver_null: Model Number: 
Sat May 27 2017 11:52:06.812 - test: driver_null: Window: 3840 0 1920 1080
Sat May 27 2017 11:52:06.812 - test: driver_null: Render Target: 1920 1080
Sat May 27 2017 11:52:06.812 - test: driver_null: Seconds from Vsync to Photons: 0.011000
Sat May 27 2017 11:52:06.812 - test: driver_null: Display Frequency: 0.000000
Sat May 27 2017 11:52:06.812 - test: driver_null: IPD: 0.063000
....
Sat May 27 2017 11:52:06.812 - test: driver_null: GetWindowBounds：x:3840, y:0, w:1920, h:1080
....

I do not know where is the problem, the logs seems OK.
I took some photos of the screens, the one showing the compositor is the primary monitor. 

![_20170527142458](https://cloud.githubusercontent.com/assets/13621537/26518710/6312a7a6-42e8-11e7-91f5-d1bb11185769.jpg)
![_20170527142506](https://cloud.githubusercontent.com/assets/13621537/26518712/6a7fe8fa-42e8-11e7-8785-96900ba0f121.jpg)
